### DISCUSSION TITLE
IntelliSense: Is it possible to suggest Aliased paths consistently?

### DISCUSSION THREAD

**Discussion by tkalmar** [vuejs/language-tools#2602] [Q&A] [answered]
Hi,
i'm using volar (take-over-mode) in an nuxt3 project. Works great so far. The only thing which is annoying me is the handling of (component) imports. Nuxt adds an Alias of `@` for the project root dir and on some places volar/vsc correctly suggests `@/components/Foo` as import but most of the time i got relative imports suggested.
Can i achive consistent imports in any way? Either use always the alias (prefered solution) or always ignore the alias and use always relative paths?
I guess volar prefers the shorter path `./baz/Foo.vue` over `@/components/foo/bar/baz/Foo.vue` but it is harder to recognize the structure with sorted imports e.g. i want to see all imports from `components/foo` together, but this isn't happening if one is imported relative and one with alias...

Thanks in advance for your great work and the fantastic project you are building

**Comment by johnsoncodehk** [comment, marked as answer]
You can try to config `"typescript.preferences.importModuleSpecifier": "non-relative"` in VSCode settings.
